subsys/mgmt/hawkbit/hawkbit.c

```diff
@@ -78,7 +78,7 @@ static bool hawkbit_initialized;
 static struct hawkbit_config {
 	int32_t action_id;
 #ifdef CONFIG_HAWKBIT_SET_SETTINGS_RUNTIME
-	char server_addr[DNS_MAX_NAME_SIZE + 1];
+	char server_addr[CONFIG_DNS_RESOLVER_MAX_QUERY_LEN + 1];
```

Comment:
hawkBit does not require the DNS_RESOLVER, so `CONFIG_DNS_RESOLVER_MAX_QUERY_LEN` could be undefined

Reply:
note this https://github.com/zephyrproject-rtos/zephyr/pull/89533

Reply:
What do you suggest be used instead of this flag? Do you want me to hardcode 255?

Reply:
> If CONFIG_DNS_RESOLVER_MAX_QUERY_LEN is defined we could use that, otherwise use INET6_ADDRSTRLEN ?

this seems like a good idea

Reply:
`#include <zephyr/net/dns_resolve.h>` can than also be removed, DNS_MAX_NAME_SIZE was the only thing we used from that include.